components/org.wso2.carbon.identity.application.authenticator.samlsso/src/main/java/org/wso2/carbon/identity/application/authenticator/samlsso/SAMLSSOAuthenticator.java

```diff
@@ -195,6 +200,32 @@ protected void processAuthenticationResponse(HttpServletRequest request, HttpSer
             stateInfoDO.setSpNameQualifier(spNameQualifier);
             context.setStateInfo(stateInfoDO);
 
+            // Add AuthnContextClassRefs received with SAML2 Response to AuthenticationContext
+            if (AS_RESPONSE.equalsIgnoreCase(context.getAuthenticatorProperties()
+                    .get(IdentityApplicationConstants.Authenticator.SAML2SSO.RESPONSE_AUTHN_CONTEXT_CLASS_REF))) {
+                if (log.isDebugEnabled()) {
+                    log.debug("AuthnContextClassRefs received with SAML response is passed to service provider.");
+                }
```

Comment:
Cant we log any context info related to sp here?

Reply:
Context-related information covered in https://github.com/wso2-extensions/identity-outbound-auth-samlsso/pull/49/files/ae21558f1877a442ff1b11a2fdbe4e1c9d7967f5#diff-2ccdf60b6f5bae54dc8690b527ec8a0fR510